site/content/3.13/graphs/graph-visualizer.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Graph Visualizer
+menuTitle: Graph Visualizer
+weight: 102
+description: >-
+  Explore your ArangoDB graphs with an intuitive and interactive interface
+---
+The **Graph Visualizer** is a browser-based tool integrated into the web interface
+of the ArangoDB Platform. It lets you explore the connections of your named graphs
+to visually understand the structure as well as to inspect and edit the attributes
+of individual nodes and edges. It also offers query capabilities and you can
+create new nodes (vertices) and relations (edges).
+
+{{< info >}}
+Graph creation is **not** performed within the Graph Visualizer. Graphs must be
```

Comment:
Would be nice to add a link to a relevant page on how to create a graph

Reply:
That documentation is specifically for the core web interface though hmm